examples/stack/sample-aws-stack-nb.json

```diff
@@ -5,11 +5,20 @@
       "id": "example-notebook",
       "service": "workspace",
       "properties": {
-        "source_path": "dev/notebook.scala",
-        "path": "/Shared/stack-cli-examples/dev/notebook",
+        "source_path": "project/notebook.scala",
+        "path": "/tmp/stack-cli-examples/workspace-example/notebook",
```

Comment:
change the path to
"/databricks/StackCliExamples/NotebookAndJobProject/helloworld.scala"